service/discovery/azure/compute.go

```diff
@@ -221,9 +221,9 @@ func (*azureComputeDiscovery) handleVirtualMachines(vm *armcompute.VirtualMachin
 				},
 				Labels: labels(vm.Tags),
 			},
-			NetworkInterface: []voc.ResourceID{},
+			NetworkInterfaces: []voc.ResourceID{},
 		},
-		BlockStorage:      []voc.ResourceID{},
+		BlockStorages:     []voc.ResourceID{},
```

Comment:
what comes to my mind though, is that actually the word `storage` has no plural form as far as I know. Should we just stick with `BlockStorage` + array? Might affect other properties as well but might be better comprehensible.

Reply:
Yes, that's true. You mean we call that "BlockStorageArray". Then I would adjust that in general.

Reply:
No I meant the name `BlockStorage` and array as data type

Reply:
For now I changed it as follows: Strings that contain the word "storage" (case insensitive) aren't converted to plural. We should decide the comment above about the plural or singular version.